src/notes/note.entity.ts

```diff
@@ -82,15 +92,19 @@ export class Note {
   // eslint-disable-next-line @typescript-eslint/no-empty-function
   private constructor() {}
 
-  public static create(owner?: User, alias?: string): Note {
+  public static create(owner?: User, alias?: string): Omit<Note, 'id'> {
```

Comment:
The undefined user is never used in the software, so it may be better to remove the `?` even from the service method `createNote`.
```suggestion
  public static create(owner: User, alias?: string): Omit<Note, 'id'> {
```

Reply:
To create anonymous notes, `owner` must be nullable

Reply:
Yes, I know but this method is never called without this argument. It is always explicitly null.

Reply:
Because we didn't create this path yet

Reply:
Yep, we will most certainly need to create Notes without an owner, so it doesn't make much sense to remove this now, only to add it later.